src/components/views/rooms/MessageComposerInputOld.js

```diff
@@ -331,11 +331,19 @@ export default React.createClass({
             return;
         }
 
-        var isEmote = /^\/me( |$)/i.test(contentText);
+        var isEmote = /^\/me( |'s|$)|my( |$)/i.test(contentText);
```

Comment:
wouldn't this catch `/me'spamming` as well? (should `'s` have a space after it?)